mersenne-31/src/complex.rs

```diff
@@ -219,7 +233,7 @@ impl Field for Mersenne31Complex<Mersenne31> {
     type Packing = Self;
 
     fn try_inverse(&self) -> Option<Self> {
-        todo!()
+        Some(self.conjugate().scale(self.magnitude().inverse()))
```

Comment:
I think we should handle the zero case as well, we could check if `self.is_zero()`, or if `self.magnitude().try_inverse()` gives `None`

Reply:
it now passes along the Option<> from M31::try_inverse